packages/applications-service-api/src/controllers/documents.v3.js

```diff
@@ -41,7 +47,59 @@ const buildFilters = (req) => ({
 	datePublishedTo: req.body.datePublishedTo
 });
 
+const documentTypeDict = {
```

Comment:
it's a style/nitpick so feel free to ignore but i was thinking it might be tidier to delegate this mapping of `type` to the 'service' functions, and keep that DB specific stuff in the service or repository?

i was thinking pass the `type` as-is to `fetchBackOfficeDocumentsByType` / `fetchNIDocumentsByType`,
and then in those functions map to the generic value to its DB equivalent